build/linux/installer/conf/fluent-bit-geneva.conf

```diff
@@ -33,6 +33,19 @@
     Record Computer ${HOSTNAME}
     Record AzureResourceId ${AKS_RESOURCE_ID}
 
+#${KubernetesFilterEnabled}[FILTER]
+#${KubernetesFilterEnabled}    Name                kubernetes
+#${KubernetesFilterEnabled}    Match               oms.container.log.la.*
```

Comment:
this has to be **geneva.container.log.***

Reply:
changed